### DISCUSSION TITLE
How to make RadioGroup options appear in horizontal alignment rather then in vertical alignment ?

### DISCUSSION THREAD

**Discussion by AkshayJangid** [cloudscape-design/components#1791] [Q&A] [answered]
I want to add two radio group button in parallel to each other. 
But due to removal of custom CSS, i'm stuck right now with button on top of each other. Is there any solution to this beside adding one one item in one radio group and creating multiple radio groups ?

**Comment by Al-Dani** [comment, marked as answer]
Hello!
Unfortunately, currently RadioGroup component supports only vertical layout. For horizontal, you can use [Tiles](https://cloudscape.design/components/tiles/) component, which has radiogroup under the hood.

**Reply by AkshayJangid** [reply]
Can i help with it ? Can i raise CR to provide this alignment ?
